app/jobs/itms/sync_all_activities_job.rb

```diff
@@ -1,7 +1,7 @@
 class Itms::SyncAllActivitiesJob < ItmsJob
-  def perform
-    Itms::Activity.find_each do |activity|
-      Itms::SyncActivityJob.perform_later(activity.itms_href)
-    end
+  def perform(downloader: ItmsJob::Downloader)
+    json = downloader.get_json_from_href('/v2/aktivita', modifiedSince: latest_timestamp)
+    hrefs = json.map { |item| item['href'] }
```

Comment:
tento `map` je tu zbytocne, rovno volajme job

Reply:
to som copypastol, je to tak vsade. chceme to zmenit vo vsetkych joboch?
(niektore tie jsony maju aj 10+MB, tipol by som ze zvysi ram usage)

Reply:
ano, upravit vsade